src/hacbs/components/PipelineRunDetailsView/tabs/PipelineRunDetailsTab.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import {
+  DescriptionList,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  DescriptionListDescription,
+  Title,
+} from '@patternfly/react-core';
+import { calculateDuration } from '../../../utils/pipeline-utils';
+
+type PipelineRunDetailsTabProps = {
+  pipelineRun: any;
```

Comment:
Should have a proper type here. But can also wait to align with the work @karthikjeeyar is doing which adds more pipeline types.

Reply:
yes, agreed but i used it temporarily... if there are more types coming let's wait on this one